contracts/base/Lenders.sol

```diff
@@ -118,6 +119,10 @@ contract Lenders is LendersInterface {
         external
         isLendingPool()
         isValid(recipient)
+        isInitialized()
+        whenNotPaused()
+        whenLendingPoolNotPaused()
+        nonReentrant()
```

Comment:
why is nonReentrant needed? It doesnt make any external function calls

Reply:
Nor this one

Reply:
The same. The change was pushed. We will review it after merging.